seed/challenges/03-front-end-libraries/react.json

```diff
@@ -714,7 +714,7 @@
         }
       },
       "tests": [
-        "getUserInput => assert(getUserInput('index').replace(/ /g, '').includes('classMyComponentextendsReact.Component{'), 'message: There should be a React component called <code>MyComponent</code>.');",
+        "getUserInput => assert(getUserInput('index').replace(/ /g, '').match(/classMyComponentextendsReact\\.Component\\s*\\{/), 'message: There should be a React component called <code>MyComponent</code>.');",
```

Comment:
The RegEx used in `match` doesn't work. 

I think it would be simpler to do:
```js
getUserInput => assert(getUserInput('index').replace(/\s/g, '').includes('classMyComponentextendsReact.Component{')
```
Where instead of replacing only space characters, we replace all whitespace with `/\s/g`

Reply:
I'm curious why you say it doesn't work.  The change was tested and proved working in my test cases-- that said, I'll update it as you request.